lib/ffmpeg/vaapi/encoder.py

```diff
@@ -66,9 +66,15 @@ def check_output(self):
     assert m is not None, "Possible incorrect profile used"
 
     # entrypoint
+    if "jpeg" == self.codec:
+      _VAEntrypoint_Encode_Mode_ = "VAEntrypointEncPicture"
+    elif self.codec in ["vp9"]:
+      _VAEntrypoint_Encode_Mode_ = "VAEntrypointEncSliceLP"
+    else :
+      _VAEntrypoint_Encode_Mode_ = "VAEntrypointEncSlice"
```

Comment:
Too confusing!

Reply:
This also breaks legacy

Reply:
Removed. Right, this is an issue, thanks finding this.